apps/qubit/modules/default/actions/fullTreeViewAction.class.php

```diff
@@ -237,15 +236,15 @@ protected function getNodeOrChildrenNodes($id, $baseReferenceCode, $children = f
     }
 
     // If not ordered in SQL, order in-memory by "text" attribute of each node
-    if (!isset($options['orderColumn']))
+    if ($options['memorySort'])
```

Comment:
This throws a PHP warning when it's not set in the options array, I know it's annoying but we should check if it's set first to avoid it, `if (isset($options['memorySort']) && $options['memorySort'])`.